src/ext/issuance.js

```diff
@@ -312,23 +312,21 @@ function verifyProofAndStoreTokens(url, tabId, tokens, issueResp, commitments) {
  * @return {Object} Object containing commitment data
  */
 function retrieveCommitments(xhr, version) {
-    let commG;
-    let commH;
-    const respBody = xhr.responseText;
-    const resp = JSON.parse(respBody);
-    const comms = resp[commitmentsKey()];
+    const resp = JSON.parse(xhr.responseText);
+    const comms = resp[getConfigName()];
+
     version = checkVersion(version);
-    if (comms) {
-        if (dev()) {
-            commG = comms["dev"]["G"];
-            commH = comms["dev"]["H"];
-        } else {
-            commG = comms[version]["G"];
-            commH = comms[version]["H"];
+    const cmt = comms[version];
+    if (typeof cmt === "undefined") {
+        throw new Error("[privacy-pass]: Retrieved version: " + version + " not available.");
+    }
+    if (version[0] === "2") {
```

Comment:
Happy to leave this in for now, but I think we should remove this version check before we merge this branch into master in the future.

Reply:
Ignore this comment, I think we can probably leave this in the initial release and then officially remove support for version `1` keys in a separate release.